Styles/components.css

```diff
@@ -228,18 +227,19 @@ body {
   cursor: pointer;
   padding: 0.4rem 0.7rem;
```

Comment:
margin, padding can be separated into separate classes and can be used throughout the application.
for example : .mt-1{margin-top : 5rem} . The class mt-1 can be used wherever needed.